governance/polls/Ratification Poll for Onboarding Growth Task Force - June 13, 2022.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Ratification Poll for Onboard Growth Task Force (MIP75c3-SP1) - June 13, 2022
+summary: Rank your preferred options for approving the Growth Task Force with the initial 1 million DAI funding, approving the task force without initial funding, and rejecting the task force. 
+discussion_link: https://forum.makerdao.com/t/mip75c3-sp1-onboard-task-force-growth-task-force/15094
+vote_type: Ranked Choice IRV
+categories:
+   - MIPs
+   - High Impact
+options:
+   0: Abstain
+   1: Approve Task Force and associated delegated authority to spend funds
+   2: Approve Task Force without associated delegated authority to spend funds
+   3: Reject Task Force
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-27T16:00:00
+---
+# Ratification Poll for Onboard Growth Task Force (MIP75c3-SP1) - June 13, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is an instant-runoff vote.**
+- **You may rank any number of options that you would support in an executive vote.**
+- **Unranked options signal that you would not support the option(s) in an executive vote.**
+- **If you have no preference to any of the listed options, you should vote 'Abstain' as your first and only choice.**
+
+## Review
+
+The community can vote in this poll to rank their support of the following options:
+* Abstain - should be the only option ranked if you have no preference on the outcome of this poll.
+* Approve Task Force and associated delegated authority to spend funds - ranking this option supports the approval of the Growth Task Force and initial funding of 1 million DAI.
+* Approve Task Force without associated delegated authority to spend funds - ranking this option supports the approval of the Growth Task Force with no initial funding.
```

Comment:
```suggestion
* Approve Task Force without associated delegated authority to spend funds - ranking this option supports the approval of the Growth Task Force with no initial funding. This would allow the task force access to the expedited governance process described in [MIP75](https://mips.makerdao.com/mips/details/MIP75).
```